canonical-websites/snapcraft.io
# Add releases page with table of revisions

Fixes https://github.com/CanonicalLtd/snap-squad/issues/599

- Adds releases page to publisher snap pages, with (react-rendered) table of revisions.
- Removes dependency on vanilla-framework-react (as there were couple of issues when using it (https://github.com/vanilla-framework/vanilla-framework-react/issues/132, https://github.com/vanilla-framework/vanilla-framework-react/issues/133)

### QA

- ./run or http://snapcraft.io-pr-822.run.demo.haus
- Sign in
- Go to any snap
- "Releases" should be in tabs navigation
- go to Releases page, list of revisions should render
<img width="1077" alt="screen shot 2018-06-27 at 16 26 19" src="https://user-images.githubusercontent.com/83575/41980475-dcce0ad8-7a26-11e8-844a-e4410e9e632a.png">
